content/_data/changelogs/weekly.yml

```
@@ -18215,30 +18215,38 @@
   - version: '2.377'
     date: 2022-11-08
     changes:
-      - type: rfe
-        category: rfe
-        pull: 7303
-        authors:
-          - car-roll
-        pr_title: Allow detached plugin location to be overridden
-        message: |-
-          Developer: Allow detached plugin location to be overridden.
       - type: rfe
         category: rfe
         pull: 7335
         authors:
           - StefanSpieker
         pr_title: Removed unused `UnbufferedBase64InputStream`
         message: |-
-          Remove deprecated and unused class <code>UnbufferedBase64InputStream</code>.
+          Removed deprecated and unused class <code>UnbufferedBase64InputStream</code>.
```

Prefer present tense rather than past tense.
```suggestion
          Remove deprecated and unused class <code>UnbufferedBase64InputStream</code>.
```